src/bilder/images/edxapp/deploy.py

```diff
@@ -113,31 +112,27 @@
         destination=Path("/var/www/.ssh/id_rsa"),
     )
 ]
+
+# Set up Consul templates
+lms_config_path = Path("/edx/etc/lms.yml")
+studio_config_path = Path("/edx/etc/studio.yml")
+forum_config_path = Path("/edx/app/forum/forum_env")
+lms_intermediate_template = Path("/etc/consul-template/templates/edxapp-lms.tmpl")
+studio_intermediate_template = Path("/etc/consul-template/templates/edxapp-studio.tmpl")
+forum_intermediate_template = Path("/etc/consul-template/templates/edx-forum.tmpl")
 consul_templates = [
-    ConsulTemplateTemplate(
```

Comment:
Why is this removed?

Reply:
The commit message describes the situation, but the idea is that by splitting the Packer builds into the Ansible run and the customization it allows us to build new instances for config changes more quickly, so that we don't have to use the 2 step consul-template rendering that we have been doing.